engines/scumm/smush/smush_player.cpp

```diff
@@ -1304,4 +1320,856 @@ void SmushPlayer::play(const char *filename, int32 speed, int32 offset, int32 st
 	CursorMan.showMouse(oldMouseState);
 }
 
+void SmushPlayer::initAudio(int samplerate, int32 maxChunkSize, int32 track1MaxSize, int32 track2MaxSize, int32 track3MaxSize, int32 track4MaxSize) {
+	char *v5;
```

Comment:
don't put not ready code to PR

Reply:
Okay, it appears I haven't commited part of the PR. Sorry for that. I'll do that and rebase pretty quick.

Reply:
I think it's better do some squash ,commits contains some commented draft code

Reply:
You are certainly right; done